### PR TITLE
Feat front items

### PR DESCRIPTION
**Pull request by mikeyrayvon**
uses group meta field on front page admin edit to select projects for front page columns

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -10,25 +10,20 @@
...
-);
+$projects = get_post_meta(get_the_ID(), '_igv_front_projects', true);
 
-// The Query
-$query = new WP_Query( $args );
+//pr($projects); die;
 
 // The Loop
-if ( $query->have_posts() ) {
+if ($projects) {
```
Comment (varffs) [inline front-page.php]: you can preserve the WP_Query here but just adding the `post__in` argument for the query from an array of the returned ids. Then the rest of the code can stay the same

**Review by varffs — CHANGES_REQUESTED** [review]
Keep using WP_Query for perf reasons

File: front-page.php (hunk 1 above)
Comment (mikeyrayvon) [reply front-page.php]: what is a "perf reason"?

**Comment by mikeyrayvon** [comment]
you have to put the callback in quotes, my dude

**Pull request merged**